### ISSUE TITLE
Data Dashboard: Add textures to the Vega maps

### ISSUE THREAD

**Issue by efregoso** [SatcherInstitute/health-equity-tracker#498]
Prior discussions with UX have included the suggestion to add more textures to the Vega maps.

Relevant Links:
- https://github.com/vega/vega/issues/1372
- https://observablehq.com/@oliviafvane/how-to-use-textures-patterns-in-vega-lite

**Comment by benhammondmusic** [comment]
We should likely be planning a way in the future to address color blindness... the subtle differences in color could easily be lost be some users and there should be another way to display these, either as an alternative or as an overlay.  There is some discussion above on fill patterns, etc. using Vega.
